tests/test_series.py

```diff
@@ -3124,13 +3133,25 @@ def test_diff() -> None:
         index_to_check_for_type=-1,
     )
     # period -> object
-    with pytest_warns_bounded(
-        RuntimeWarning, "overflow encountered in scalar multiply"
-    ):
+    if WINDOWS:
+        with pytest_warns_bounded(
+            RuntimeWarning, "overflow encountered in scalar multiply"
+        ):
+            check(
+                assert_type(
+                    pd.Series(
+                        pd.period_range(start="2017-01-01", end="2017-02-01", freq="D")
+                    ).diff(),
```

Comment:
If you use `pd.Series(pd.period_range(start="10:00", end="18:00", freq="h")).diff()`, then we can avoid the Windows issue.  Can you use that test instead?

Reply:
Never mind.  Even with `freq="h"`, the same issue occurs.

Reply:
do you prefer to keep this test as is (with the pytest_warns_bounded) or shall I just remove it ?

Reply:
keep as is